stac_check/lint.py

```diff
@@ -33,6 +36,36 @@ def __post_init__(self):
         self.file_name = os.path.basename(self.item).split('.')[0]
         self.best_practices_msg = self.create_best_practices_msg()
 
+    def parse_config(self, config_file):
+        default_config = {
+            'linting': {
+                'searchable_identifiers': True, 
+                'percent_encoded': True,
+                'item_id_file_name': True,
+                'catalog_id_file_name': True,
+                'check_summaries': True,
+                'null_datetime': True,
+                'check_unlocated': True,
+                'check_geometry': True,
+                'bloated_links': True,
+                'bloated_metadata': True,
+                'check_thumbnail': True,
+                'links_title': True,
+                'links_self': True,
+            }
+        }
```

Comment:
Could we load the default config from `stac-check.config.yml` to ensure the real defaults stay in-sync with the example configuration file?

Reply:
True yes - good idea